cmd/find/entry.go

```diff
@@ -0,0 +1,47 @@
+package cmdfind
+
+import (
+	"github.com/puppetlabs/wash/api/client"
+	apitypes "github.com/puppetlabs/wash/api/types"
+)
+
+type entry struct {
+	*apitypes.Entry
```

Comment:
I'm curious why this is a pointer. Not clear it's used that way.

Reply:
Oh, so we avoid doing a bunch of copy-by-values, which I think may be bad for hundreds of thousands of entries. Can change it back if you prefer the non-pointer version.

Reply:
I don't think it's actually avoiding those copies in lots of cases, and it's a small object so I wouldn't worry about it in the first place.

Reply:
Updated.